src/Benchmarks/Benchmarks.csproj

```diff
@@ -1,34 +1,33 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.Web">
-
-  <Import Project="..\..\build\common.props" />
-
+<Project Sdk="Microsoft.NET.Sdk.Web">
   <PropertyGroup>
     <TargetFramework>netcoreapp2.0</TargetFramework>
     <OutputType>Exe</OutputType>
+    <MvcRazorCompileOnPublish>true</MvcRazorCompileOnPublish>
```

Comment:
To enable pre-compilation and to be explicit about it. Means we need fewer changes when using 1.1.x because pre-compilation wasn't the default with those versions.